modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc

```diff
@@ -735,8 +750,12 @@ and show only active users.
 
 Here is what the full filter can look like.
 
+[source]
 ----
-&(|(objectclass=organizationalPerson))(!(userAccountControl:1.2.840.113556.1.4.803:=2))(|(|(memberof=CN=Domain Users,CN=Users,DC=dp,DC=mosreg,DC=ru)(primaryGroupID=513))))
+&(|(objectclass=organizationalPerson))
+  (!(userAccountControl:1.2.840.113556.1.4.803:=2))
+  (|(|(memberof=CN=Domain Users,CN=Users,DC=dp,DC=mosreg,DC=ru)(primaryGroupID=513)))
+)
 ----
```

Comment:
You may want to improve readbility if possible:
```
  (|
    (|
      (memberof=CN=Domain Users,CN=Users,DC=dp,DC=mosreg,DC=ru)
      (primaryGroupID=513)
    )
  )
```

Reply:
I updated some of the examples so that there would be no wrapping, but hadn't intended to update all of them. I'll take another look. Thanks for the recommendation.